.travis.yml

```diff
@@ -1,5 +1,13 @@
 language: bash
 
+sudo: false
+
+install:
+  - nvm install 6
+  - nvm use 6
+  - npm install -g purescript@0.10.2 pulp bower
+
 script:
   - bin/fetch-configlet
   - bin/configlet .
+  - bin/build.sh
```

Comment:
Should this be called test.sh rather than build.sh? (I don't actually have strong feelings about the name of the file, but if I were looking for the tests I'd be looking for something with _test_ in the name).